### PR TITLE
fix eth packages typing issue, not export default #3198

### PR DESCRIPTION
**Pull request by ariesjia**
## Description

Please include a summary of the changes and be sure to follow our [Contribution Guidelines](../CONTRIBUTIONS.md).

typing issue, export is not correct.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [  ] I ran `npm run dtslint` with success and extended the tests and types if necessary.
- [x] I ran `npm run test:unit` with success.
- [x] I ran `npm run test:cov` and my test cases cover all the lines and branches of the added code.
- [x] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [x] I have tested my code on the live network.
- [x] I have checked the Deploy Preview and it looks correct.
- [  ] I have updated the `CHANGELOG.md` file in the root folder.

### CODE REVIEW

File: packages/web3-eth-ens/package.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
     },
     "devDependencies": {
         "dtslint": "^3.4.1",
-        "typescript": "^3.9.5"
+        "typescript": "^3.9.5",
+        "web3-eth": "1.3.0"
```
Comment (Spacesai1or) [inline packages/web3-eth-ens/package.json:30]: Should this (`web3-eth`) be a dependency?

**Comment by Spacesai1or** [comment]
So I'm a little confused on why we would want to do something like:

```javascript
import Contract, { ContractOptions} from 'web3-eth-contract';
```

If we update the packages, in this case `web3-eth-contract`, to have the default export be an object containing all the exports such as:

```javascript
export default {
    Contract,
    ContractOptions
}
```

You would still be able to do:

```javascript
import { Contract, ContractOptions} from 'web3-eth-contract';
```

Am I mistaken, is there a reason you wouldn't want to do it that way?

**Comment by ariesjia** [comment]
@spacesailor24 
The current version the declaration tell us should use  `import { Contract, ContractOptions } from 'web3-eth-contract';` , but the  `Contract` is got `undefined`
Because source code is
```js
module.exports = Contract;
```
for ES module it equals  `export default`.  So the typing is not match the source.
If we use  `import  Contract, { ContractOptions } from 'web3-eth-contract';`, we will got the correct object ,  but  we will get Typescript compile err due to the declaration.

If we change the export will bring API broken change， this commit is only for quick fix

**Review by BryanLaura — APPROVED** [review]

**Review by BryanLaura — APPROVED** [review]
Fix #3732

**Review by BryanLaura — COMMENTED** [review]
`export` `import` `default `

**Pull request closed without merging**